.github/workflows/ci.yml

```diff
@@ -23,13 +23,19 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: "Create env file"
+        run: |
+          echo POLYGON_PRIVATE_KEY=${{ secrets.POLYGON_PRIVATE_KEY }} >> .env
```

Comment:
I think we should be running the tests in Mumbai instead of Matic so we don't waste MATIC for every test run.

Reply:
Unless we setup the tests to run in a fork so we don't actually execute in the live network.

Reply:
The tests do run in a fork as off now. But the env variables still seem to not work